server/pulp/server/webservices/views/repositories.py

```diff
@@ -938,11 +937,11 @@ def post(self, request):
             invalid_criteria.add_child_exception(e)
             raise invalid_criteria
 
-        regeneration_tag = tags.action_tag('content_applicability_regeneration')
-        async_result = regenerate_applicability_for_repos.apply_async_with_reservation(
-            tags.RESOURCE_REPOSITORY_PROFILE_APPLICABILITY_TYPE, tags.RESOURCE_ANY_ID,
-            (repo_criteria.as_dict(),), tags=[regeneration_tag])
-        raise pulp_exceptions.OperationPostponed(async_result)
+        async_result = ApplicabilityRegenerationManager.regenerate_applicability_for_repos(
```

Comment:
This got me thinking about the name of the function being called. I suggest ``queue_regenerate_applicability_for_repos``, since all it really does it make some tasks. That also makes it more obvious that it's going to return some kind of async job reference.